### ISSUE TITLE
Primary and Secondary Buttons

### ISSUE THREAD

**Issue by gnakm** [department-of-veterans-affairs/roadrunner#73]
![a1024a22-d55b-11e5-80bf-37705a20bfbf](https://cloud.githubusercontent.com/assets/13420618/13305357/c8f611e4-db28-11e5-9265-00771be1230d.png)

**Comment by webinista** [comment]
@gnakm Is this design ready to go? Can you provide the hex colors?

**Comment by gnakm** [comment]
Emily did this but I'll take a stab at it.
Primary Green Button = #2e8540
Secondary Button Border = #323a45
Secondary Button Text = #0071bc 

The button focus for both should be the same one used in the web standards: https://playbook.cio.gov/designstandards/buttons/

**Comment by emilyville** [comment]
Oh! please use standard colors that are variables defined elsewhere. I
didn't think of this as "final". :)

I.e. what @gnkm said.

**Comment by webinista** [comment]
I think it makes sense to go with what's already been defined by the USWDS and the `.usa-button-outline` class for the secondary button.

![screen shot 2016-02-26 at 8 49 50 am](https://cloud.githubusercontent.com/assets/354277/13359403/f26195e4-dc68-11e5-8c49-6a5c905d9836.png)

**Comment by gnakm** [comment]
Yeah I used all the standard colors minus the grey. @webinista Thanks for going ahead with this!

**Comment by gnakm** [comment]
<img width="503" alt="screen shot 2016-03-08 at 1 08 41 pm" src="https://cloud.githubusercontent.com/assets/13420618/13611135/ef776254-e52e-11e5-9604-4da1be506099.png">

Content:

Primary: 

Apply
With subtle rounded edge and friendly color, the Vets.gov green button is approachable to all users who are looking to learn more about eligibility and apply for various benefits. Designers and writers are advised to use actionable words so that users can anticipate the next steps to a process. Green buttons are commonly used to apply for benefits and can found at the [top right corner](https://www.vets.gov/disability-benefits/claims-process/) or [bottom left corner](https://www.vets.gov/disability-benefits/) of the body copy, depending on other page elements such as piano keys or side navigation.

<img width="174" alt="screen shot 2016-03-08 at 1 13 18 pm" src="https://cloud.githubusercontent.com/assets/13420618/13611261/9d4da280-e52f-11e5-9c2b-43823adfdfd5.png">

Search, Sign In
For clear and actionable steps, the primary blue button is intended to help users navigate throughout different sections and tools on vets.gov. It can be found on forms, tools and sign-in sections. 

<img width="179" alt="screen shot 2016-03-08 at 1 13 24 pm" src="https://cloud.githubusercontent.com/assets/13420618/13611259/984b0a34-e52f-11e5-950e-a7427a9b50d9.png">

Default color: 008BBC
Hover and Active color: 006F97

Secondary:
While the primary button has a stronger weight and bolder color, the secondary button uses the white space to build contrast for a user’s task. The Vets.gov secondary button is commonly used in forms or pages with multiple actionable items. 

<img width="175" alt="screen shot 2016-03-08 at 1 13 30 pm" src="https://cloud.githubusercontent.com/assets/13420618/13611257/93f2ac3a-e52f-11e5-846f-d81b2d81ec08.png">

Code can be found in secondary button section: https://playbook.cio.gov/designstandards/buttons/

**Comment by webinista** [comment]
Confused about whether to include a code sample on the page or just link to the USWDS Play Book. Which is it?

**Comment by emilyville** [comment]
I guess we could just link for now, if nothing has changed? I don't have strong thoughts or feelings though.

**Comment by gnakm** [comment]
@webinista I should have clarified. I meant to say to include a code sample that is found on the playbook. Could we put it in a box?

**Comment by webinista** [comment]
Sure. So for clarity's sake: if we want to use a code sample, we also have to use HTML examples instead of images. The USWDS examples have "live" HTML and some JavaScript that automatically generates the code sample. This is not a big deal to do. But it does involve building / copying-and-pasting HTML examples.

**Comment by gnakm** [comment]
@emilyville @webinista Ah I see. 

How about for now we link for code snippets that way we have things consistent especially since we will be using images like playbook? We could use this sprint to get as much on the site and keep it consistent and then develop it as we go.
